wiki/Performance_points/th.md

```diff
@@ -0,0 +1,122 @@
+# Performance points
+
+**Performance points** (หรือเรียกสั้น ๆ ว่า **pp**) ที่มีจุดมุ่งหมายในการแสดงการพัฒนาของผู้เล่นใน osu!
+
+มันมีจุดมุ่งหมายที่จะแสดงความสามารถของผู้เล่น โดยจะดูจากเวลาที่ใช้เล่นไปทั้งหมด จนถึงการคำนวณความสามารถจริง ๆ ของผู้เล่น คซึ่งค่านี้จะนำมาจากการคำนวณคะแนนที่ขึ้นอยู่กับระดับความยากและคะแนนที่ผู้เล่นได้ใน [บีทแมพ(beatmap)](/wiki/Beatmaps) นั้น ๆ
+
+## ประวัติ
+
+ระบบนี้มีการนำมาใช้แบบเปิดเผยครั้งแรกในช่วงปี 2012 ซึ่งในตอนนั้นคนรู้จักระบบนี้ในนามว่า โปรเจค *'???'* เท่านั้น และระบบนี้ก็มีการตั้งชื่อไม่นานหลังจากนั้น
+
+หลังจากนั้น ระบบนี้ก็เป็นที่รู้จักกันในนามว่า pp (ที่ย่อมาจาก "performance points") ระบบใหม่นี้จุดมุ่งหมายที่จะเปลี่ยนการนับคะแนนจากการดู[คะแนน](/wiki/Score)รวมเฉย ๆ เป็นสิ่งที่แสดงความสามารถที่แท้จริงของผู้เล่น ซึ่งหลังจากการเปิดตัว ระบบนี้ก็ได้รับการตอบรับอย่างดีจากฐานผู้เล่นในระยะเวลานั้น
+
+ไม่กี่เดือนหลังจากการเปิดตัว ระบบนี้ก็ได้มีการมาใช้อย่างเต็มที่ในอัปเดต osu! 20120722-24่ เพื่อนำมาใช้แทนระบบคะแนน [Ranked](/wiki/Beatmaps#ranked) แบบเก่า คะแนนใหม่จะถูกคำนวณทุก ๆ 30 นาที และหลังจากนั้นในเดือนสิงหาคม ได้มีการพัฒนาให้มีการอัปเดตแบบ real-time
+
+*โน้ต: ppv1 คือระบบ Performance Point แบบแรก ซึ่งมีบันทึกการเปลี่ยนแปลงที่สามารถพบเจอได้ที่ [กระทู้นี้](https://osu.ppy.sh/community/forums/topics/92185)*
+
+ระบบดังกล่าวมีการใช้งานมามากกว่า 1 ปี จนกระทั่ง [Tom94](https://osu.ppy.sh/users/1857058) ผผู้พัฒนาระบบคำนวณคะแนน *osu!tp* ได้เข้าร่วม [ทีมงาน osu!](/wiki/People/The_Team) และได้นำระบบของเขามาใช้ และมีการตั้งชื่อระบบนี้ว่า *ppv2* และมีการเปิดตัวให้ใช้เมื่อวันที่ 27 มกราคม 2557 และระบบเก่าก็ถูกเปลี่ยนชื่อเป็น *[ppv1](/wiki/Performance_points/ppv1)*
+ppv2 กำลังถูกใช้งานอยู่ขณะนี้ ซึ่งการอัปเดตเกี่ยวกับระบบนี้จะถูกประกาศไว้ที่ [บันทึกการเปลี่ยนแปลง](https://osu.ppy.sh/p/changelog?category=pp)
+
+## การคำนวณ
+
+การคำนวณ Performance points จะขึ้นอยู่กับระดับความยากของบีทแมพ ซึ่งจะถูกคำนวณโดยอัลกอริทึมที่จะแตกต่างกันไปในแต่ละ [เกมโหมด](/wiki/Game_mode).
+
+ระดับความยากของบีทแมพที่ผู้เล่น เล่นอยู่จะเป็นตัวกำหนด pp ที่ได้ตอนท้าย ตามการออกแบบสูตรคำนวณจะใช้ 4 ค่าหลัก ซึ่งประกอบไปด้วย **[การเล็ง](#การเล็ง)** **[ความเร็ว](#ความเร็ว)** **[ความแม่นยำ](#ความแม่นยำ)** และท้ายสุด **[ความเครียด](#ความเครียด)** ซึ่งหลังจากนั้นค่าทุกอย่างจะถูกนำมารวมกันและ จะแสดงผลลัพธ์โดยรวมที่จะขึ้นอยู่กับ [ระดับความยาก](/wiki/Difficulties) และประสิทธิภาพของผู้เล่นในแมพนั้นนั้น
```

Comment:
This wasn't overlooked on my part. Apologies.

...และท้ายสุด **[ความเครียด](#ความเครียด)**... -> ...และ **[ความเครียด](#ความเครียด)**...